app/parsers/bulkrax/csv_parser.rb

```diff
@@ -116,7 +116,7 @@ def create_from_importer
     end
 
     def create_from_collection
-      work_ids = ActiveFedora::SolrService.query("member_of_collection_ids_ssim:#{importerexporter.export_source}").map(&:id)
+      work_ids = ActiveFedora::SolrService.query("member_of_collection_ids_ssim:#{importerexporter.export_source}", rows: 2_000_000_000).map(&:id)
```

Comment:
I think this is the right play, but whenever I see it all I can think is:
![](https://media.tenor.com/images/faf934d304adf3abe163e3a6d192c178/tenor.gif)